### PR TITLE
Fix #294 (CI) - I can't see all the attributions

### PR DESCRIPTION
**Pull request by gianpaj**
Fixed issue: https://github.com/clearlydefined/website/issues/294

When you click on the **Attributions** value it shows a Popover with all the Copyrights in both the Definition and Full Details View (modal).
You can scroll horizontally and vertically.

### CODE REVIEW

**Comment by gianpaj** [comment]
I've fixed issues Simone reported to me as well as:
- added a Popover for Discovered Licenses (both in the Full Details View and Component list pages)
- fixed #296 (Filelist - Cannot add copyrights when none are present)

ready for another review

**Comment by storrisi** [comment]
Not currently approved since some regression bugs showed up

File: src/components/PopoverRenderer.js [hunk 1]
```diff
@@ -64,15 +64,14 @@ class PopoverRenderer extends Component {
...
       item && (
         <div key={`${item.value}_${index}`} className="popoverRenderer__items">
-          <div
-            className={`popoverRenderer__items__value ${item.isDifferent && 'popoverRenderer__items__value--isEdited'}`}
-          >
+          <div className={`popoverRenderer__items__value ${extraClass}}`}>
             {
               <InlineEditor
-                extraClass={item.isDifferent ? 'popoverRenderer__items__value--isEdited' : ''}
+                extraClass={extraClass}
```
[the remark this answers is not in the document]
Comment (gianpaj) [reply src/components/PopoverRenderer.js]: It's to remove duplicate code as we were doing this twice:
```js
item.isDifferent ? 'popoverRenderer__items__value--isEdited' : ''
```

**Comment by gianpaj** [comment]
I made the necessary changes as discussed

**Pull request merged**